README.md

```diff
@@ -20,18 +20,25 @@ $ pip install -i https://test.pypi.org/simple/ --extra-index-url https://pypi.or
 ## Features
 
 1.  `missing_detect`: This function will take in a dataframe and generate a table listing the number of missing values and the percentage of missing values for each column. 
-2.  `missing_impute`: This function will take in a dataframe and genereate a imputated dataframe based on some simple imputation method. The imputation methods can also be customized by the user.
+2.  `missing_impute`: This function will take in a dataframe and generate a imputed dataframe based on some simple imputation method. The imputation methods can also be customized by the user.
```

Comment:
an imputed

Reply:
Good catch! I fixed those and forgot to commit.